Readme.md

```diff
@@ -11,10 +11,10 @@ A Swift library for UI automation of apps and browsers via communication with [W
 A `swift-webdriver` "Hello world" using `WinAppDriver` might look like this:
 
 ```swift
-let session = Session(
+let session = try Session(
     webDriver: WinAppDriver.start(), // Requires WinAppDriver to be installed on the machine
     desiredCapabilities: WinAppDriver.Capabilities.startApp(name: "notepad.exe"))
-session.findElement(locator: .name("close"))?.click()
+try session.requireElement(locator: .name("close")).click()
```

Comment:
why not just change `findElement` to return a non optional?

Reply:
1. A goal of mine is for the API to support the common scenario of negative queries without having to handle thrown errors. i.e. "don't use errors for control flow". It also has the benefits of being less noisy if you're debugging with first-chance exceptions
2. The name `findElement` is neutral and I'm trying to call out that `if try? findElement() == nil` is a bad idea because it will wait the full timeout in the expected scenario. There might be other ways to achieve this, curious for your thoughts

Reply:
Reviewing other languages bindings, I think you're right it makes more sense to throw:
- https://www.browserstack.com/guide/findelement-in-selenium
- https://toolsqa.com/selenium-webdriver/c-sharp/findelement-and-findelements-commands-in-c/